DOC-1913, DOC-1776, and DOC-1954: GCP Fixes (#2441)
RedisLabs/redislabs-docs: content/rc/security/private-service-connect.md

Now that you have a pending Private Service Connect entry, you need to create the endpoints in your application's VPC:

> 1. Follow Google Cloud's guide to [enable Cloud DNS](https://cloud.google.com/dns/docs/set-up-dns-records-domain-name) for your Google Cloud project if you haven't already.

Reviewer: ```suggestion
1. If you have not already done so, [enable Cloud DNS](https://cloud.google.com/dns/docs/set-up-dns-records-domain-name) for your Google Cloud project.
```
